data-es2016plus.js

```diff
@@ -5776,6 +5776,39 @@ exports.tests = [
           reactnative0_70_3: true,
           rhino1_7_13: false
         }
+      },
+      {
+        name: 'resolving identifier in parent scope',
+        exec: function () {/*
+          {
+            let a = ["hello world"];
+            class MyClass {
+              c = a[(0)]; 
```

Comment:
are the parens needed here to trigger the issue?

Reply:
Yes, the parentheses are important. The JIT bug only occurs when parenthesis are included in the initializer.

Reply:
A comment to that affect on this line would be good, to ensure nobody removes them in the future.

Reply:
Added :+1: